Content.Server/GameTicking/Rules/SurvivorRuleSystem.cs

```diff
@@ -0,0 +1,121 @@
+﻿using Content.Server.Administration.Logs;
+using Content.Server.Antag;
+using Content.Server.GameTicking.Rules.Components;
+using Content.Server.Mind;
+using Content.Server.Roles;
+using Content.Server.Shuttles.Systems;
+using Content.Shared.Database;
+using Content.Shared.GameTicking.Components;
+using Content.Shared.Mind;
+using Content.Shared.Mobs.Components;
+using Content.Shared.Mobs.Systems;
+using Content.Shared.Survivor.Components;
+using Content.Shared.Tag;
+using Robust.Server.GameObjects;
+
+namespace Content.Server.GameTicking.Rules;
+
+public sealed class SurvivorRuleSystem : GameRuleSystem<SurvivorRuleComponent>
+{
+    [Dependency] private readonly RoleSystem _role = default!;
+    [Dependency] private readonly MindSystem _mind = default!;
+    [Dependency] private readonly IAdminLogManager _adminLog = default!;
+    [Dependency] private readonly AntagSelectionSystem _antag = default!;
+    [Dependency] private readonly TransformSystem _xform = default!;
+    [Dependency] private readonly EmergencyShuttleSystem _eShuttle = default!;
+    [Dependency] private readonly TagSystem _tag = default!;
+    [Dependency] private readonly MobStateSystem _mobState = default!;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<SurvivorRoleComponent, GetBriefingEvent>(OnGetBriefing);
+    }
+
+    // TODO: Planned rework post wizard release when RandomGlobalSpawnSpell becomes a gamerule
+    protected override void Started(EntityUid uid, SurvivorRuleComponent component, GameRuleComponent gameRule, GameRuleStartedEvent args)
+    {
+        base.Started(uid, component, gameRule, args);
+
+        var allHumans = _mind.GetAliveHumans();
+
+        foreach (var human in allHumans)
+        {
+            if (!human.Comp.OwnedEntity.HasValue)
+                continue;
+
+            var ent = human.Comp.OwnedEntity.Value;
+
+            // Alive Humans does get the mindcomp, however this has better logic to get the mindId
+            // No need to repeat the same code
+            if (!_mind.TryGetMind(ent, out var mind, out _) || HasComp<SurvivorComponent>(ent))
+                continue;
```

Comment:
```suggestion
            if (HasComp<SurvivorComponent>(ent))
                continue;
```
`human` is equal to `mind`, so no need to get it a second time
I would recommend renaming the variable so it is more clear